README.md

```diff
@@ -411,12 +411,12 @@ Arguments:
   [HOST]  Name of the host to log in to
 
 Options:
-  --username TEXT  Username of the remote user  [required]
+  --email TEXT     Email of the remote user  [required]
   --password TEXT  Password of the remote user  [required]
   --help           Show this message and exit.
 
 Example:
-  qne login --username=my_user_name --password=my_password https://api.quantum-network.com
```

Comment:
Not for this repo but we should change and possibly change the descriptions on the QNE-ADK website when we release the new adk. Maybe add a ticket for this to find and adjust all the references to username on the website